joomla/joomla-cms
# [4.0] Move csp configuration to the com_csp component

### Summary of Changes

Move csp configuration to the com_csp component

### Testing Instructions

configure the http headers using com_csp's options make sure all is still working as expected

### Expected result

csp configs are just moved and all features still work

### Actual result

csp config is in the plugin but we now have a component so the options get moved

### Documentation Changes Required

none

cc @SniperSister
